### DISCUSSION TITLE
Machine Learning

### DISCUSSION THREAD

**Discussion by LilacHo** [UndefiedCse/Heat-Island#22] [General]
Share any thought or update

**Comment by vismaya-v** [comment]
Can you suggest a better Algorithm?
I have a dataset which has 29 inputs and 1 output. Currently the training dataset consists of 2242 rows, I have tried a couple of algorithms and the best I got was with Gaussian process regression exponential GP2(R^2=0.50) . Can you suggest a better algorithm.
I have attached the dataset herewith. I appreciate any help.
[Data.xlsx](https://github.com/UndefiedCse/Heat-Island/files/13748435/Data.xlsx)
